docs/source/Tutorial.md

```diff
@@ -1575,6 +1570,10 @@ a bit more flexibility.
   monster_builder.add_equipped(axe.Union());
   auto orc = monster_builder.Finish();
 ~~~
+**Important**: Unlike structs, you should not interlace creation of tables or other objects,
```

Comment:
Please use `nest` instead of `interlace`. `interlace` is more like "alternate", which doesn't imply hierarchy.

Reply:
This change was deliberate. "Nest" is currently used two times in the tutorial. First:

> Note: To create a vector of nested objects (e.g. tables, strings, or other vectors), collect their offsets into a temporary data structure, and then create an additional vector containing their offsets.

Second (this one):

> Important: Unlike structs, you should not nest tables or other objects, which is why we created all the strings/vectors/tables that this monster refers to before start. If you try to create any of them between start and end, you will get an assert/exception/panic depending on your language.

To me, those two usages refer to different concepts. First is about nesting of _data_. Second is about nesting of _builders_. If my reasoning is correct, I'd propose to leave "interlace" (or any other term different from "nest"). If it is not, I'd appreciate an explanation why.

Reply:
To me they refer to the same concept. Like I said, I don't believe "interlace" expresses hierarchy.

Reply:
Is the issue specific to a hierarchy? What if I "nest" building of other tables, that will become a part of other unrelated object? Will this still be illegal?

Reply:
Yes, this is about hierarchy, building a buffer is a very hierarchical activity. While the data appears in-order, it needs to be constructed pre-order.